options.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html PUBLIC '-//W3C//DTD XHTML 1.0 Strict//EN'
+  'http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd'>
+<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'>
+<body>
+	What should happen if forbidden address is detected?<br><br>
```

Comment:
Osobiście bardziej bym dał coś w stylu "Choose a way to block sites". Bo to forbidden sites, to trochę jak do filtru rodzicielskiego na strony pornograficzne.

Reply:
to "blocked address"